views/Admin.react.js

```diff
@@ -739,6 +745,30 @@ class AdminPage extends Component {
                 dataKey="access"
                 width={this.state.width / 6}
               />
+              <Column
+                label="Account Expiration"
+                cellRenderer={({ cellData, rowData: { uid } }) => (
+                  console.log({ cellData, uid }),
+                  (
+                    <TextField
+                      id="account_expiration"
+                      type="date"
+                      defaultValue={moment(cellData).format('YYYY-MM-DD')}
+                      InputLabelProps={{
+                        shrink: true,
+                      }}
+                      onChange={(e) =>
+                        this.updateAccountExpiration({
```

Comment:
do we need this to be a method on the component? What about calling `updateUser` directly? Also how are we handling the result of the API call to show the new data in the cell?

Reply:
The text field updates automatically when the date is selected